ros2doctor/ros2doctor/api/platform.py

```diff
@@ -24,37 +26,38 @@ def print_platform_info():
     platform_name = platform.system()
     # platform info
     print('PLATFORM INFORMATION')
-    print('system               : ', platform_name)
-    print('Platform Info        : ', platform.platform())
+    print_term('system', platform_name)
+    print_term('platform info', platform.platform())
     if platform_name == 'Darwin':
-        print('Mac OS version       : ', platform.mac_ver())
-    print('release              : ', platform.release())
-    print('processor            : ', platform.processor())
+        print_term('mac OS version', platform.mac_ver())
+    print_term('release', platform.release())
+    print_term('processor', platform.processor())
 
     # python info
     print('PYTHON INFORMATION')
-    print('version              : ', platform.python_version())
-    print('compiler             : ', platform.python_compiler())
-    print('build                : ', platform.python_build())
+    print_term('version', platform.python_version())
+    print_term('compiler', platform.python_compiler())
+    print_term('build', platform.python_build())
```

Comment:
This is definitely an improvement. I still think it would be good if the `reports` just returned information and the command was responsible for outputting it.

Reply:
I was thinking about making that change as a feature in the next PR. Like you suggested last time, I'd like to add arguments like `--report_all` or `--report_failed` to let users control what they would like to see based on the check results.